modules/apps/headless/headless-discovery/headless-discovery-web/webpack.config.js

```diff
@@ -12,11 +12,11 @@
  * details.
  */
 
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+//const HtmlWebpackPlugin = require('html-webpack-plugin');
+//const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
-const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin');
-const StyleExtHtmlWebpackPlugin = require('style-ext-html-webpack-plugin');
+//const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin');
+//const StyleExtHtmlWebpackPlugin = require('style-ext-html-webpack-plugin');
```

Comment:
Innocent people are going to die because of this commented out code...

![humanity](https://user-images.githubusercontent.com/7074/99388491-f8aa7d80-28d5-11eb-99f6-3bd3b980dad2.gif)

(Not really; just hope that Brian doesn't reject this as a result).

Reply:
I commented that for @nhpatt to have a look at it, but in theory that commit should be replaced by a new one fixing it. If we finally decide to ship this broken I will remove the comments.

Reply:
removed the comments